### DISCUSSION TITLE
Faulty by Strategy

### DISCUSSION THREAD

**Discussion by ednt** [rustdesk/rustdesk#5142] [Feature Request]
If the Name is too long for a Strategy, the 3 Dots are moved to the left side and unreachable.
In this case, no modification is possible!

**Comment by dinger1986** [comment]
Can you open these on https://github.com/rustdesk/rustdesk-server-pro/issues please and thanks!

**Comment by rustdesk** [comment]
@21pages

**Reply by 21pages** [reply]
fixed
